github_connector/README.rst

```diff
@@ -0,0 +1,106 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+======================================
+Load Github Data in your Odoo Instance
+======================================
+
+This module allow you to
+
+* recover social informations from Github. (Members of organizations,
+  Pull Requests, Issues, Comments, ...)
+* download source code from Github
+
+Configuration
+=============
+
+Once installed, you have to:
+* Open your openerp.cfg file and add extra settings to mention github
+ credencials:
```

Comment:
Thanks for your remarks.
Do not hesitate to make a PR directly on github for such minor changes. It's faster.
Kind regards.